docs/package.json

```diff
@@ -62,8 +62,13 @@
     "@types/react": "^17.0.2",
     "autoprefixer": "^10.2.4",
     "eslint-config-next": "12.0.4",
+    "globby": "^13.1.1",
+    "json2md": "^1.12.0",
     "json5-loader": "^4.0.1",
     "postcss": "^8.2.6",
+    "react-docgen-typescript": "^2.2.2",
+    "react-markdown": "^8.0.2",
+    "remark-gfm": "^3.0.1",
```

Comment:
There is an older version of remark-gfm ^1.0.0 declared in our dependencies for docs already, should we update this?

Reply:
Good catch. I also found some never used code for the previous installed `remark-gfm`, also removed them